contracts/interfaces/STETHInterface.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+interface STETHInterface {
```

Comment:
We need to use the wstETH interface here, which doesn't include getPooledEthByShares.